lib/policy_machine_storage_adapters/in_memory.rb

```diff
@@ -28,6 +28,10 @@ class InMemory
       # The results are paginated via will_paginate using the pagination params in the params hash
       # The find is case insensitive to the conditions
       define_method("find_all_of_type_#{pe_type}") do |options = {}|
+        if options.delete(:where)
```

Comment:
make sense but can you clarify "If you'd like, you could make this more limited (an attribute_containing key or something like that) and I could add it to the other adapters" ? I understand not wanting to expose activerecord functionality for that reason, but shouldn't it be up to the consumers (i.e. dalton) to manage this if it is useful to have such functionality?